### PR TITLE
Add fromAddress different from delegate

### PR DESCRIPTION
**Pull request by puccca**

### CODE REVIEW

**Comment by dakk** [comment]
GJ, please refactor to make it optional (if no fromAddress set, then use delegate address)

**Comment by puccca** [comment]
Hey! It is optional already. If fromAddress is not set, it will paty from delegate's address (lines 240-241 from liskpool3.py).

File: liskpool3.py [hunk 1]
```diff
@@ -235,19 +235,24 @@ def paymentCommandForLiskCore(conf, address, amount):
 def savePayments(conf, topay):
 	addr = r(conf, 'accounts?username=' + conf['delegateName'])['data'][0]['summary']['address']
 	binAddress = addressToBinary(addr)
+	
+	binFromAddress = addressToBinary(conf['fromAddress'])
```
Comment (dakk) [inline liskpool3.py:239]: the check should be done before calling addressToBinary only 
```python
if 'fromAddress' in conf:
   binAddress = ...
````

File: README.md [hunk 2]
```diff
@@ -20,7 +20,8 @@ Fork this repo; edit config.json and modify the first lines with your settings:
 	"poolState": "poollogs.json",                               // Where to save pool state
 	"paymentsFile": "payments.sh",                              // Where to save payments commands
 	"includeSelfStake": false,                                  // True if we want to include selfstake in distribution calculations
-	"multiSignature": false										// True if you're using a multisig account
+	"multiSignature": false,				    // True if you're using a multisig account
+	"fromAddress": ""					    // Address from where to make the payments. Unchanged if the payments will be made from the delegate's address
```
Comment (dakk) [inline README.md:24]: // Address from where to make the payments. Remove if the payments will be made from the delegate's address

File: config.json [hunk 3]
```diff
@@ -9,5 +9,6 @@
 	"poolState": "poollogs.json",
 	"paymentsFile": "payments.sh",
 	"includeSelfStake": false,
-	"multiSignature": true
+	"multiSignature": true,
+	"fromAddress": ""
```
Comment (dakk) [inline config.json:13]: Remove from exaple file; the field should be populated or not present

**Review by dakk — CHANGES_REQUESTED** [review]

**Review by dakk — APPROVED** [review]

**Pull request merged**